config/parse.go

```diff
@@ -334,6 +333,23 @@ func resolveHandlers(c *Config) error {
 	return nil
 }
 
+func resolveTransformers(c *Config) error {
+	logger := confLog.WithField("method", "resolveTransformers")
+	for transformerName, t := range skogul.TransformerMap {
+		logger = logger.WithField("transformer", transformerName)
+
+		if c.Transformers[t.Name] != nil {
+			logger.Debug("Using predefined transformer")
+		} else {
+			logger.Error("Unknown transformer")
+			return skogul.Error{Source: "config", Reason: fmt.Sprintf("Unknown transformer %s", t.Name)}
+		}
+
+		t.T = &c.Transformers[t.Name].Transformer
```

Comment:
I don't think you need to dereference this? We need to for handlers, but that's because a handler is a type, not an interface. Not sure though...? Just notice we're not doing it for senders.